### PR TITLE
added Junit create book it's work in-progress

### PR DESCRIPTION
**Pull request by kmohanraj**

### CODE REVIEW

**Comment by araviind0009** [comment]
Good work

**Review by araviind0009 — APPROVED** [review]
Looks good.

File: src/main/java/com/example/learningspring/model/Book.java [hunk 1]
```diff
@@ -14,13 +14,13 @@
   @NotEmpty(message = "Author Name can not be empty")
   private String authorName;
   private String description;
-  private Boolean published;
+  private String published;
```
Comment (araviind0009) [inline src/main/java/com/example/learningspring/model/Book.java:17]: Why this column is changed to String?

File: src/test/java/com/example/learningspring/controller/BookControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+  @Autowired
+  private MockMvc mockMvc;
+
+  @MockBean
+  private BookService bookService;
+
+  private static ObjectMapper mapper = new ObjectMapper();
+
+  @Test
+  public void createBook_whenPostMethod() throws Exception {
```
Comment (araviind0009) [inline src/test/java/com/example/learningspring/controller/BookControllerTest.java:48]: Method name should be consistent

**Review by araviind0009 — CHANGES_REQUESTED** [review]

File: src/main/java/com/example/learningspring/model/Book.java (hunk 1 above)
Comment (kmohanraj) [reply src/main/java/com/example/learningspring/model/Book.java:17]: just need to show the display like **active** or **in-active**

File: src/test/java/com/example/learningspring/controller/BookControllerTest.java (hunk 2 above)
Comment (kmohanraj) [reply src/test/java/com/example/learningspring/controller/BookControllerTest.java:48]: The name was modified as consistent

**Pull request merged**